diff --git a/helm/image-distribution-operator/templates/_helpers.tpl b/helm/image-distribution-operator/templates/_helpers.tpl
index ab9bf4c..16a1c94 100644
--- a/helm/image-distribution-operator/templates/_helpers.tpl
+++ b/helm/image-distribution-operator/templates/_helpers.tpl
@@ -23,6 +23,7 @@ helm.sh/chart: {{ .Chart.Version | quote }}
 app.kubernetes.io/name: {{ include "chart.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
 {{- end }}
 
 
